get_age.py

```diff
@@ -1,21 +1,43 @@
 import sys
 import datetime
+import os
+import time
+import platform
 
-year = int(input("What year were you born? "))
-month = int(input("What month were you born? "))
-day = int(input("What day were you born? "))
+def clear():
+	if platform.system() == "Linux":
+		os.system("clear")
+	elif platform.system() == "Windows":
+		os.system("cls")
+	else:
+		pass
 
-yes_no = input("So you were born on %02d/%02d/%04d. Is that correct? [yes/no]" % (day, month, year))
-if yes_no.lower() == "no":
-    print("Too bad!")
-else:
-    print("Excellent!")
-
-    today = datetime.date.today()
-    age = today.year - year
-    if month > today.month:
-        age -= 1
-    elif month == today.month and day > today.day:
-        age -= 1
-
-    print("You are", age, "years old")
\ No newline at end of file
+def ask():
+    year = int(input("What year were you born? "))
+    month = int(input("What month were you born? "))
+    day = int(input("What day were you born? "))
+    yes_no = input("So you were born on %02d/%02d/%04d. Is that correct? [yes/no]" % (day, month, year))
+    if yes_no.lower() == "no":
+        print("Too bad!")
+        clear()
+        ask()
+    else:
+        print("Excellent!")
+        time.sleep(1)
+        today = datetime.date.today()
+        age = today.year - year
+        if month > today.month:
+           age -= 1
+        elif month == today.month and day > today.day:
+            age -= 1
+    clear()
+    print("You are", age, "years old")
+    continue = input("Do you want to continue ? [Y/N] ")
+    if continue == "Y" or "y" or "yes":
+		clear()
+		ask()
+    else:
+		sys.exit(0)
```

Comment:
You don't really need to sys.exit(). (I've almost never used this). The program will simply drop off the bottom. There is a pattern which some people use which is:

`if __name__ == '__main__': sys.exit(ask())`

which allows the leading function (in this case, `ask`) to return a value indicating an error. In our case there's no particular point in this as we won't do anything with it outside the program